### ISSUE TITLE
[CW-1974] Forwarding an email message from a conversation 

### ISSUE THREAD

**Issue by aerostovtsev** [chatwoot/chatwoot#6673]
Hi! 
Email source is connected to Chatwoot - hello@example.com. How is it possible to implement the scenario of resending email from a conversation. Details:

1. Customer customer@example.com send a message to the mailbox hello@example.com - ok
2. A new conversation is created. The message contains information - From, To, Subject, content and attachments - ok
3. It is required from the Chatwoot interface to send email (The message that initiated the creation of this conversation) with all its data to finance@exampe.com

How to implement step 3 in Chatwoot interface? Subtitles do not contain all the information.

<sub>[CW-1974](https://linear.app/chatwoot/issue/CW-1974/forwarding-an-email-message-from-a-conversation)</sub>

**Comment by sanjayws** [comment]
I think you can use the send transcript function or add "finance" into the conversation as "join conversation" if that's what you're looking for

**Comment by aerostovtsev** [comment]
Yes, we tried to implement it this way, but it's inconvenient. The transcription does not contain email attachments. Employee from finance@example.com does not work in the Chatwood app and does not plan to.
